apps/expo/src/components/tracker/workout-timer.tsx

```diff
@@ -0,0 +1,23 @@
+import { useEffect, useState } from 'react'
+import { Text, View } from 'react-native'
+
```

Comment:
timer works nicely, I do have this when we start doing rep counting backward 

https://www.npmjs.com/package/react-native-countdown-circle-timer

Reply:
thats the one I also was planning on using